cmd/finch/nerdctl.go

```diff
@@ -313,6 +346,50 @@ func resolveIP(host string, logger flog.Logger) string {
 	return host
 }
 
+type volume struct {
+	original    string
+	destination string
+}
+
+func reverseSortVolumes(volumes []string) []string {
+	volumeSlices := make([]volume, 0)
```

Comment:
opt: could preallocate memory here to length of volumes

Reply:
Tried it before. One of our linter is not happy with that. https://github.com/ashanbrown/makezero

Reply:
Weird, TIL.